CONTRIBUTING.md

```diff
@@ -36,7 +36,7 @@ If you have any questions that _aren't_ answered by this contribution guide, let
 
 #### **Do you want to contribute a template to the Template Gallery?**
 
-* Please read the Template Gallery [contribution guide](content/templates/CONTRIBUTING.md).
+* Please read the Template Gallery [contribution guide](https://github.com/victoriabernard92/template-registry/CONTRIBUTING.md).
```

Comment:
this link appears to be broken - should be https://github.com/victoriabernard92/template-registry/blob/master/CONTRIBUTING.md, right?

can we get the template registry repo brought into the cloudflare org before we merge this PR?